samples/browser/build.gradle

```diff
@@ -132,6 +132,8 @@ dependencies {
 
     servoImplementation project(':browser-engine-servo')
 
+    implementation project(':service-sync-places')
```

Comment:
For sync-logins we have a separate sample app. We'd likely not add all sync features to the sample browser, I think. So, we'd have a sync specific sample app and the reference-browser (having all features.)